.changeset/rotten-tables-brush.md

```diff
@@ -0,0 +1,5 @@
+---
+"@firebase/database": patch
+---
+
+Fixed issue where `get()` updated incorrect cache.
```

Comment:
nit: Fixed issue where `get()` updated cache incorrectly for non-default queries.